code/modules/urist/datums/uristchem.dm

```diff
@@ -21,3 +21,26 @@
 /datum/reagent/drink/pilk/affect_ingest(mob/living/carbon/M, var/alien, var/removed)
 	..()
 	M.adjustBrainLoss(0.1)
+
+/datum/chemical_reaction/napalk
+	name = "Napalk"
+	result = /datum/reagent/drink/napalk
+	required_reagents = list(/datum/reagent/drink/pilk = 1, /datum/reagent/napalm = 1)
+	result_amount = 2
+	mix_message = "The pilk mixes with the gooey napalm, forming a slick slurry of a substance that reeks of milk and gasoline. This is a crime against humanity."
+
+/datum/reagent/drink/napalk
+	name = "Napalk"
+	description = "A horrible flurry of substance."
+	taste_description = "curdled milk and gasoline"
+	reagent_state = LIQUID
+	color = "#76685f"
+
+	glass_name = "Napalk"
+	glass_desc = "A glass of napalk, a combination of napalm and pilk. What compells you to drink this?"
+	glass_special = list(DRINK_VAPOR)
+
+/datum/reagent/drink/napalk/affect_ingest(mob/living/carbon/M, var/alien, var/removed)
+	..()
+	M.adjustToxLoss(0.5)
+	M.adjustBrainLoss(0.2)
```

Comment:
Brainloss is kind of a weird effect here. Why not make this work like a diet version of Unholy Water that doesn't have a pass-through on cultists?